src/http.rs

```diff
@@ -123,6 +124,23 @@ where
                 })
                 .context(CallRequestSnafu)?;
 
+            let (parts, body) = response.into_parts();
+            let mut data_buf: Option<Vec<u8>> = Some(Vec::new());
+
+            let body = body
+                .map_data(move |data| {
+                    if let Some(buf) = &mut data_buf {
+                        buf.extend_from_slice(&data);
+                        if buf.len() >= 20 {
```

Comment:
I believe this will need to be followed up with a change to make the size configurable. 20 bytes is going to be much too small for most situations. I'd think 100 might be a better default.

Reply:
Do we want to limit the number of bytes? As a user, I think I'd want the full request. Agreed we could at least make this configurable with an option to disable the limit.